lib/src/commands/packages/commands/check/commands/licenses.dart

```diff
@@ -21,7 +21,13 @@ class PackagesCheckLicensesCommand extends Command<int> {
     Logger? logger,
     PubLicense? pubLicense,
   })  : _logger = logger ?? Logger(),
-        _pubLicense = pubLicense ?? PubLicense();
+        _pubLicense = pubLicense ?? PubLicense() {
+    argParser.addFlag(
+      'ignore-failures',
+      help: 'Avoids terminating whenever a license fails to be retrieved.',
```

Comment:
nit: but i would just make it say something like `Ignore any license that failed to be retrieved` otherwise the help does not match the parameter imho